packages/config/src/cookieTransform.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+const jws = require('jws');
+
+function cookieTransform(proxyReq, req) {
+    const cookie = req.headers.cookie;
+    const match = cookie && cookie.match(/cs_jwt=([^;]+);/);
+    if (match) {
+        const cs_jwt = match[1];
+        const { payload } = jws.decode(cs_jwt);
+        const identity = {
+            entitlements: {
+                insights: { is_entitled: true },
+                smart_management: { is_entitled: true },
+                openshift: { is_entitled: true },
+                hybrid: { is_entitled: true },
+                migrations: { is_entitled: true },
+                ansible: { is_entitled: true }
```

Comment:
I think we should make these objects configurable. If somebody wants to use different identity than "super user".

Reply:
added customization